src/utils/FormatDate.js

```diff
@@ -0,0 +1,23 @@
+export function FormatDate() {
+    const months = {
+        0: 'January',
```

Comment:
This utility is already available in the utils folder to get the latest date try modifying that to format the date and in case no parameter is passed use the latest date else use the parameter passed.

Take the latest pull from develop and verify all the redundant code that you have written. The date and month  array constants are also present in the constants folders